static/js/cookie-consent.js

```diff
@@ -17,10 +17,10 @@ $(window).on('load', function(){
     }
   });
 });
-// Use UTC to determine if the time zone resides in or around the EU (within UTC -1 to +6)
+// Use UTC to determine if the time zone resides in or around the EU (within UTC 0 to +4 - https://www.timeanddate.com/time/zones/eu)
 function testEUtimezone(){
```

Comment:
Could you add another (optional) check to force the EU behavior, say if the query string includes ="cookie-dialog=eu" so we can try this without changing timezones?

Reply:
I forgot that I added them but there are two commands that you can run from the console to toggle either the consent or the notice:
toggleconsent()
togglenotice()

Will that work?

Reply:
that works!